web/pgadmin/tools/import_export/__init__.py

```diff
@@ -142,42 +157,62 @@ def index():
     return bad_request(errormsg=_("This URL cannot be called directly."))
 
 
-def _get_ignored_column_list(data, driver, conn):
+def _get_force_quote_column_list(data, driver, conn):
     """
-    Get list of ignored columns for import/export.
+    Get list of required columns for import/export.
     :param data: Data.
+    :param key: Key.
     :param driver: PG Driver.
     :param conn: Connection.
-    :return: return ignored column list.
+    :return: return required column list.
     """
-    icols = None
+    cols = None
 
-    if data['icolumns']:
-        ignore_cols = data['icolumns']
+    # if export is using query then we need to check * is available in the
+    # force_quote_columns then return *.
+    if ('force_quote_columns' in data and 'is_query_export' in data and
+            data['is_query_export'] and '*' in data['force_quote_columns']):
```

Comment:
all three ifs has check for `'force_quote_columns' in data`. Can be clubbed in one if and others will go inside to simplify.